impeller/entity/contents/content_context.h

```diff
@@ -185,11 +185,13 @@ struct ContentContextOptions {
   CompareFunction stencil_compare = CompareFunction::kEqual;
   StencilOperation stencil_operation = StencilOperation::kKeep;
   PrimitiveType primitive_type = PrimitiveType::kTriangle;
+  bool has_stencil_attachment = true;
```

Comment:
I think this is fine (especially for non-TBDR archs), but we should be careful when adding more options to this struct as that open up another dimension along which PSO variants can be created (albeit with the same shaders). I'm somewhat uncomfortable we need to do this today already but we already have the prototypes and they are created concurrently. So it works out.

Perhaps we should add a comment to this struct explaining the consequences of adding fields here?

Reply:
Done. Added a docstring explaining that complicated Flutter apps could easily be expected to form hundreds of PSOs in total, but should never reach the order of e.g. 10s of thousands.

BTW, we'll need another param to set the color attachment pixel format for wide gamut (I suspect this is the major mismatch issue that @gaaclarke has been running into).